pages/public_cloud/ai_machine_learning/notebook_tuto_16_mlflow/guide.en-gb.md

```diff
@@ -0,0 +1,119 @@
+excerpt: Learn how to use MLflow for model tracking and versioning
+updated: 2023-10-18
+---
+
+## Objective
+
+The aim of this tutorial is to show you how to integer [**MLflow**](https://mlflow.org/) inside AI Notebooks in order to track and compare your Machine Learning models.
+
+**MLflow** is an open-source platform for *Machine Learning workflows management*. You can use this tool for ML model tracking, versioning but also storage and deployment.
+
+- **How to use MLflow inside AI Notebooks?**
+
+![MLflowOverview](images/mlflow-inside-notebook.png){.thumbnail}
+
+For this purpose, we will compare **image classification** models: [EfficientNets](https://paperswithcode.com/method/efficientnet).
+
+> [!primary]
+>
+> EfficientNet models are pre-trained on [ImageNet](https://www.image-net.org/) dataset.
+>
+
+In this context, several EfficientNet models will be trained on the same dataset ([stanford_dogs](https://www.tensorflow.org/datasets/catalog/stanford_dogs#:~:text=The%20Stanford%20Dogs%20dataset%20contains,training%20and%208580%20for%20testing.)). The **Stanford Dogs** dataset contains images of 120 breeds of dogs from around the world.
+
+The different models will be tracked and compared thanks to **MLflow** in order to be compare their performance:
+
+- EfficientNet-b0
+- EfficientNet-b1
+- EfficientNet-b2
+- EfficientNet-b3
+- EfficientNet-b4
+- EfficientNet-b5
+- EfficientNet-b6
+- EfficientNet-b7
+
+## Requirements
+
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
+- An AI Notebooks project created inside a [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
+- A user for AI Notebooks
+
+## Instructions
+
+You can launch the notebook from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB) or via the ovhai [CLI](/pages/public_cloud/ai_machine_learning/cli_11_howto_run_notebook_cli).
+
+### Launching a Jupyter notebook with "TensorFlow" via UI (Control Panel)
+
+To launch your notebook from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), refer to the following steps.
+
+#### Code editor
+
+Choose the `Jupyterlab` code editor.
+
+#### Framework
+
+In this tutorial, the `TensorFlow` framework is used. You can choose the `2.12` version: `tf2.12-py311-cudaDevel11.8`.
+
+#### Resources
+
+Using GPUs is recommended to train the **EfficientNet** models.
+
+> [!primary]
+>
+> Here, using `1 GPU` is sufficient.
+>
+
+### Launching a Jupyter notebook with "TensorFlow" via CLI
+
+*If you do not use our CLI yet, follow [this guide](/pages/public_cloud/ai_machine_learning/cli_10_howto_install_cli) to install it.*
+
+If you want to launch your notebook with the OVHcloud AI CLI, choose the `jupyterlab` editor and the `tensorflow` framework.
```

Comment:
If you do not explain how to launch a Jupyter notebook with Tensorflow via UI, maybe it would be better to avoid saying 'if you want' since the user doesn't have 2 options

```suggestion
To launch your notebook with the OVHcloud AI CLI, choose the `jupyterlab` editor and the `tensorflow` framework.
```